lib/money/currency.rb

```diff
@@ -1,14 +1,13 @@
+require "money/currency/loader"
+
 class Money
   class Currency
-    class UnknownCurrency < ArgumentError; end
-
-    CURRENCY_DATA = "#{File.expand_path('../../../config', __FILE__)}/currency_iso.json".freeze
+    extend Loader
```

Comment:
Why extend the Loader? I mean, you could just do ` @@currencies ||= Loader.load_currencies`.